dolphinscheduler-ui-next/src/views/datasource/list/detail.tsx

```diff
@@ -141,7 +141,6 @@ const DetailModal = defineComponent({
                 rules={rules}
                 ref='detailFormRef'
                 require-mark-placement='left'
-                label-placement='left'
                 label-width={180}
```

Comment:
`label-width` is not required in top-down forms.

Reply:
ok，i will fix it